test/algo/nas/test_space_hub_oneshot.py

```diff
@@ -190,6 +195,9 @@ def test_hub_oneshot(space_type, strategy_type):
             pytest.skip('The space has used unsupported APIs.')
     if strategy_type in ['darts', 'gumbel'] and space_type == 'mobilenetv3':
         pytest.skip('Skip as it consumes too much memory.')
+    
+    if space_type == "autoformer" and strategy_type != "random":
+        pytest.skip('The AutoFormer sapce only support randomoneshot currently.')
```

Comment:
Are you planning to support algos other than random in this PR? It looks pretty straightforward to me...

Also, you mis-spelled "space"...